OnTimeIssueFolderMakerPlant/OnTime/OnTimeValueResolver.cs

```diff
@@ -18,13 +18,25 @@ public class OnTimeValueResolver
 
     public Tuple<int, string> ResolveOnTimeValue(string headerValue)
     {
-      return this.ResolveNewOntime(headerValue) ?? this.ResolveOldOntime(headerValue);
+      return this.ResolveOntime17(headerValue) ?? this.ResolveNewOntime(headerValue) ?? this.ResolveOldOntime(headerValue);
     }
 
     #endregion
 
     #region Methods
 
+    private Tuple<int, string> ResolveOntime17(string bufferValue)
```

Comment:
This method looks like a copy of the `ResolveNewOntime` one with the difference in the delimiter. Wouldn't it be more appropriately to just introduce one more parameter for the delimiter and re-use the `ResolveNewOntime` method for both `NewLine` and `\t` delimiters?